baker/siteRenderers.tsx

```diff
@@ -677,6 +679,19 @@ export const renderReusableBlock = async (
     return cheerioEl("body").html() ?? undefined
 }
 
+export const renderExplorerIndexPage = async (
+    knex: KnexReadonlyTransaction
+): Promise<string> => {
+    const explorersBySlug = await getPublishedExplorersBySlug(knex)
+    const explorers = Object.values(explorersBySlug).sort((a, b) =>
```

Comment:
Doesn't really matter here, but in general it's usually better for perf to do the sorting in the DB in the same query.